pkg/test/framework/components/echo/deployment/builder.go

```diff
@@ -302,6 +303,10 @@ func (b *builder) getOrCreateNamespace(prefix string) (*builder, namespace.Insta
 func (b *builder) deployServices() (err error) {
 	services := make(map[string]string)
 	for _, cfg := range b.configs {
+		if b.ctx.Settings().EnableDualStack && cfg.IPFamilyPolicy == "" {
+			cfg.IPFamilies = "IPv6, IPv4"
+			cfg.IPFamilyPolicy = string(corev1.IPFamilyPolicyRequireDualStack)
```

Comment:
wondering if we should just always set PreferDualStack and not be conditional at all?

Reply:
Will that mask any dualstack errors? Ideally not, but just asking

Reply:
I think it makes better sense to explicitly RequireDualStack to make sure the apps are really running dualstack services?

Reply:
If you're explicitly running `dualstack` _tests_, and your cluster doesn't support `dualstack`, the _tests_ should just fail (`RequireDualStack`), rather than silently fallback (`PreferDualStack`).


So yeah - `Require` I think.

Reply:
I mostly agree and am fine with this.

Reply:
I do want to get to a world where everything is just dual stack but this probably makes the most sense in the current context